sdk/python/tests/expediagroup/test_milvus_online_store.py

```diff
@@ -394,12 +357,66 @@ def test_milvus_update_delete_unavailable_collection(self, repo_config, caplog):
             assert len(utility.list_collections()) == 0
 
     def test_milvus_online_write_batch(self, repo_config, caplog):
+        self._create_collection_in_milvus(self.collection_to_write, repo_config)
+
+        feature_view = FeatureView(
+            name=self.collection_to_write,
+            source=SOURCE,
+        )
 
         total_rows_to_write = 100
+        data = self._create_n_customer_test_samples_milvus(n=total_rows_to_write)
+        MilvusOnlineStore().online_write_batch(
+            config=repo_config, table=feature_view, data=data, progress=None
+        )
 
-        data = self.create_n_customer_test_samples_milvus(n=total_rows_to_write)
+        with MilvusConnectionManager(repo_config.online_store):
+            collection = Collection(name=self.collection_to_write)
+            progress = utility.index_building_progress(collection_name=collection.name)
+            assert progress["total_rows"] == total_rows_to_write
 
-        # Creating a common schema for collection to directly add to Milvus
+    def test_milvus_teardown_with_non_empty_collection(self, repo_config, caplog):
+        self._create_collection_in_milvus(self.collection_to_write, repo_config)
```

Comment:
Does it make sense to test teardown on a FV with an empty collection or one that was already dropped, at least to confirm idempotence on multiple calls?

Reply:
yes, makes total sense. I'll add them

Reply:
@omirandadev I've added a test with an empty collection and one where the collection doesn't exist